packages/utils/src/lib/reports/md/details.ts

```diff
@@ -13,7 +13,6 @@ export function details(
 <summary>${title}</summary>
 
 ${content}
-
```

Comment:
:warning: The blank line is needed to ensure Markdown in `content` is rendered correctly.

If you're worried about extra lines, use `content.trim()` instead.

Reply:
Added it back! Thx.